easybuild/easyconfigs/r/R/R-3.1.3-goolf-1.7.20-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,386 @@
+    ('reshape2', '1.4.1', ext_options),
+    ('proto', '0.3-10', ext_options),
+    ('munsell', '0.4.2', ext_options),
+    ('labeling', '0.3', ext_options),
+    ('scales', '0.2.4', ext_options),
+    ('ggplot2', '1.0.1', ext_options),
+    ('Hmisc', '3.15-0', ext_options),
+    ('fastcluster', '1.1.16', ext_options),
+    ('chron', '2.3-45', ext_options),
+    ('data.table', '1.9.4', ext_options),
+    ('igraph', '0.7.1', ext_options),
+    ('GeneNet', '1.2.12', ext_options),
+    ('ape', '3.2', ext_options),
+    ('htmltools', '0.2.6', ext_options),
+    ('RJSONIO', '1.3-0', ext_options),
+    ('caTools', '1.17.1', ext_options),
+    ('gplots', '2.16.0', ext_options),
+    ('ROCR', '1.0-5', ext_options),
+    ('httpuv', '1.3.2', ext_options),
+    ('R6', '2.0.1', ext_options),
+    ('shiny', '0.11.1', ext_options),
+    ('adegenet', '1.4-2', ext_options),
+    ('phylobase', '0.6.8', ext_options),
+    ('adephylo', '1.1-6', ext_options),
+    ('animation', '2.3', ext_options),
+    ('bigmemory.sri', '0.1.3', ext_options),
+    ('bigmemory', '4.4.6', ext_options),
+    ('calibrate', '1.7.2', ext_options),
+    ('clusterGeneration', '1.3.4', ext_options),
+    ('raster', '2.3-33', ext_options),
+    ('dismo', '1.0-12', ext_options),
+    ('expm', '0.99-1.1', ext_options),
+    ('extrafontdb', '1.0', ext_options),
+    ('Rttf2pt1', '1.3.3', ext_options),
+    ('extrafont', '0.17', ext_options),
+    ('fields', '8.2-1', ext_options),
+    ('shapefiles', '0.7', ext_options),
+    ('fossil', '0.3.7', ext_options),
+    ('geiger', '2.0.3', ext_options),
+    ('glmnet', '1.9-8', ext_options),
+    ('labdsv', '1.6-1', ext_options),
+    ('MatrixModels', '0.4-0', ext_options),
+    ('stabs', '0.5-1', ext_options),
+    ('mboost', '2.4-2', ext_options),
+    ('msm', '1.5', ext_options),
+    ('nor1mix', '1.2-0', ext_options),
+    ('np', '0.60-2', ext_options),
+    ('polynom', '1.3-8', ext_options),
+    ('polspline', '1.1.9', ext_options),
+    ('TH.data', '1.0-6', ext_options),
+    ('multcomp', '1.4-0', ext_options),
+    ('gridExtra', '0.9.1', ext_options),
+    ('rms', '4.3-0', ext_options),
+    ('RWekajars', '3.7.12-1', ext_options),
+    ('RWeka', '0.4-24', ext_options),
+    ('slam', '0.1-32', ext_options),
+    ('tm', '0.6', ext_options),
+    ('TraMineR', '1.8-9', ext_options),
+    ('chemometrics', '1.3.9', ext_options),
+    ('FNN', '1.1', ext_options),
+    ('ipred', '0.9-4', ext_options),
+    ('knitr', '1.9', ext_options),
+    ('statmod', '1.4.20', ext_options),
+    ('miscTools', '0.6-16', ext_options),
+    ('maxLik', '1.2-4', ext_options),
+    ('mlogit', '0.2-4', ext_options),
+    ('getopt', '1.20.0', ext_options),
+    ('gsalib', '2.1', ext_options),
+    ('reshape', '0.8.5', ext_options),
+    ('optparse', '1.3.0', ext_options),
+    ('klaR', '0.6-12', ext_options),
+    ('neuRosim', '0.2-12', ext_options),
+    ('affyio', '1.34.0', bioconductor_options),
+    ('BiocInstaller', '1.18.3', bioconductor_options),
```

Comment:
most R 3.1.3 easyconfigs are using BiocInstaller 1.16.2, from Bioconductor 3.0

This has been bumped to 1.16.3, see http://bioconductor.org/packages/3.0/bioc/html/BiocInstaller.html

1.18.3 is from a more recent Bioconductor 'release' though, so maybe we should resort to 1.16.3 in here (and sync up the other R v3.1.3 easyconfigs)?